### PR TITLE
feat(route): Start on support for generic discourse forums.

### PR DESCRIPTION
**Pull request by dzx-dzx**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/discourse/0/posts
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/advanced/script-standard) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/advanced/script-standard)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 full article
  - [ ] 使用缓存 Use cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/advanced/pub-date) [date and time](https://docs.rsshub.app/en/joinus/advanced/pub-date)
  - [x] 可以解析 Parsed
  - [ ] 时区正确 Correct time zone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Missing en docs which sits under `website/i18n/en/docusaurus-plugin-content-docs/current`

File: website/docs/routes/bbs.md [hunk 1]
```diff
@@ -142,6 +142,19 @@ import Route from '@site/src/components/Route';
...
+## Discourse
+
+使用前需设置环境变量`DISCOURSE_CONFIG_{id}`为
+```
+{
+    "link": link,
+    "key": key
+}
+```
+的JSON序列化字符串.
```
[the remark this answers is not in the document]
Comment (dzx-dzx) [reply website/docs/routes/bbs.md:154]: I saw that it said the list was incomplete, so I thought I should keep it that way...

**Comment by dzx-dzx** [comment]
> Missing en docs which sits under `website/i18n/en/docusaurus-plugin-content-docs/current`

What should I do in this file…?

**Review by TonyRL — COMMENTED** [review]
Please add details about the environment variable in both `website/docs/install/README.md`(English) and `website/i18n/zh/docusaurus-plugin-content-docs/current/install/README.md`(Chinese).

You don't need to add the format of the env in `bbs.md`

**Pull request merged**